src/snowflake/snowpark/_internal/open_telemetry_prototype.py

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+import inspect
+from typing import Dict
+
+# this parameter make sure no error when open telemetry is not installed
+open_telemetry_not_found = False
+try:
+    from opentelemetry import trace
+    from opentelemetry.sdk.resources import SERVICE_NAME, Resource
```

Comment:
Not used, can be removed

Reply:
I still need the trace to set current span right?

Reply:
Comment was for `from opentelemetry.sdk.resources import SERVICE_NAME, Resource`